# HG changeset patch
# User jwe@segfault.lan
# Date 1201917115 18000
#      Fri Feb 01 20:51:55 2008 -0500
# Node ID e4bb408219343f557da6a063b5acce27f76f9422
# Parent  e759d01692dbb69ff28b6d70940239d93ff77a9d
Added tag ss-2-1-53 for changeset e759d01692db

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -1,2 +1,3 @@
 d56e0d75e7c9760f2a3e750c2d71a5c52bb76b98 ss-2-1-45
 dece11da64ed86ac2347f48ffa13fd046bef7de3 ss-2-1-52
+e759d01692dbb69ff28b6d70940239d93ff77a9d ss-2-1-53
